internal/config/testdata/inline_table.toml

```diff
@@ -0,0 +1,15 @@
+[[outputs.http]]
```

Comment:
Simplify all input configurations down to the bare minimum to exercise the issue.  This will reduce confusion about if the behavior is dependent on other aspects of the file.